### ISSUE TITLE
Finish glibxx quest

### ISSUE THREAD

**Issue by springmeyer** [mapnik/node-mapnik#835]
The `glibxx quest` is an endeavor to build mapnik mason binaries against libstdc++ 4.9 (with this additional patch enabled (https://github.com/mapnik/mapnik/pull/3806).

Currently these are the tasks remaining to get this quest working with node-mapnik:

GLIBCXX_3.4.20 is still showing up in the binaries for node-mapnik. The question is where it is coming from. My hunch is that it is coming from the boost libraries and it will go away if we pull in boost libraries that were built with libstdc++-4.8-dev because throw_out_of_range_fmt was introduced in 4.9. So I think this should be fixable by:

 - [x] changing the mapnik mason package to build against boost 1.65.1 which is built against 4.8 per mapbox/mason#409 - done in https://github.com/mapbox/mason/commit/4ed4662eab91f5d7e40ed24c1de8f4a04acad9b7
 - [x] triggering a new mapnik v3.0.17 build
 - [x] changing install_mason.sh to point to boost 1.65.1 as well

previously part of https://github.com/mapnik/node-mapnik/pull/824

**Comment by springmeyer** [comment]
After updating to boost 1.65.1 I'm still seeing:

```
0.34s$ ./scripts/check_glibcxx.sh
checking ./lib/binding/mapnik.node
                 U std::__throw_out_of_range_fmt(char const*, ...)@@GLIBCXX_3.4.20
```

Which means to fix this we'll either need to:

 - downgrade to libstdc++4.8-dev (though I don't want to do this due to https://github.com/mapnik/node-mapnik/issues/760, https://github.com/mapnik/node-mapnik/issues/759, and https://github.com/mapnik/node-mapnik/issues/761)
 - or apply the same fix we used for mapnik core to node-mapnik: https://github.com/mapnik/mapnik/blob/master/src/glibc_workaround.cpp

**Comment by springmeyer** [comment]
Solved by #838
